### ISSUE TITLE
Admin: Dashboard

### ISSUE THREAD

**Issue by jlyon** [proudcity/pattern#27]
- [x] Add seeclickfix map to dashboard
- [x] Add questions needing to be answered

**Comment by jlyon** [comment]
From Luke

Re: http://pattern.getproudcity.com/?v=&page=dashboard

Didn't know best place/repo to put this, but wanted to pass along. Think it would be good to have some sort of human touch to the dashboard, even if it's just a 'Proudly serving #city#' header. Just feel like we have an opportunity to add personality/inspiration and tie in the brand, make it less like work. Maybe even that adorable photo of Alex and his cat.

Related:

Slack’s in-app communications are intended to reinforce the idea it’s designed for you, a regular human being, by other normal people. When you log in for the first time, a dialog box pops up to say, "Hello. Thank you for signing up for Slack. We’re really happy to have you!" In the morning, you might be greeted with cute little phrases such as, "Please enjoy Slack responsibly."

"Slack’s tone might not be a benefit for every kind of company," Butterfield acknowledges. "But even people who work in the most boring parts of the most boring accounting firms are still human beings. Some of them still go home and smoke weed, for example—not that smoking weed is the hallmark of humanity."

http://www.fastcodesign.com/3050294/innovation-by-design/slacks-workplace-revolution?utm_source

**Comment by jlyon** [comment]
Questions @aschmoe @lukefretwell 
- [x] Eval #7 
- [x] How to make it more personal
- [x] How to better-highlight PC help/blog content
- [x] Other blocks people might want to add
- [x] How this should look for different roles (user stories?).  How do roles differ for 10k pop city vs 100k city vs 1M city:
  - author - Mayoral staffer, just posts press releases/events to mayor's subsite
  - editor - Mayor chief of staff, has publishing permissions
  - admin - CIO/tech dude

**Comment by jlyon** [comment]
Based on #7:

http://pattern.getproudcity.com/?v=&page=dashboard1
http://pattern.getproudcity.com/?v=&page=node-add1
http://pattern.getproudcity.com/?v=&page=node-add2

Questions:
- Do we like top/side flow?  Are the items correct for each?
- For content edit, do we want a sidebar?
- Should topnav be black or gray?

**Comment by lukefretwell** [comment]
Nice work, @jlyon!
- I like the sidebar. Not 100% on the actual elements/aesthetic/text, but in general think this is solid. Huge potential to highlight various services. 
- I like the cleanness of no sidebar on content edit.
- Topnav black seems better. At least something that differentiates from sidenav.

**Comment by aschmoe** [comment]
Yeah I think http://pattern.getproudcity.com/?v=&page=node-add1 actually looks great.

A nice deviation from standard drupal interface.  I think keeping that admin quick bar super simple is key to ease of use.  

I like the content edit region a little smaller on my external desktop, but on a laptop screen the edit area is looking a little cramped... we may need to look at some solutions for that.

**Comment by jlyon** [comment]
@aschmoe so you like a sidebar on the content edit page?  I think I'm with @lukefretwell

> I like the cleanness of no sidebar on content edit.

Thoughts on the blocks on the dashboard? 
- What do we think about the "quick tips" "from the PC blog" sections on the left?
- Anything else that we should show on the default dashboard?

**Comment by aschmoe** [comment]
> so you like a sidebar on the content edit page?

Guess I don't really care but we need to include a way to open menu panel.

> What do we think about the "quick tips" "from the PC blog" sections on the left?

Think that's helpful... could also be  "Latest from community" aka how to / answered question from forums / slack.

**Comment by jlyon** [comment]
Re: Slack: Created issue to look into that in proud (Drupal distro) repo: https://github.com/proudcity/proud/issues/37

**Comment by aschmoe** [comment]
- [x] split side menu into sections eg: setup gets 311, social, agencies
- [x] Implement text changes (see screenshot)
- [x] Add Analytics
- [x] Add Answers

![file 6](https://cloud.githubusercontent.com/assets/1634686/10236423/1be5a74e-685a-11e5-850f-37d56ee69b15.png)

**Comment by aschmoe** [comment]
@lukefretwell @jlyon Made the edits from the call yesterday.

http://pattern.getproudcity.com/?v=&page=admin-dashboard1#

**Comment by jlyon** [comment]
I like it. I think maybe no icons for the submenu items (under Settings).

**Comment by aschmoe** [comment]
@jlyon removed.

**Comment by jlyon** [comment]
from luke:
Seeing this on phone.

Comments:
- feel like we need to make 311,  events, jobs top-level. 
- move users below analytics 
- use book or something different for content

Sorry for doing this via email.

**Comment by jlyon** [comment]
What goes in 311? Most of that is under Settings/Answers.

Added Events, Jobs. Moved Appearance under Settings to try to keep this list from becoming too long.

**Comment by jlyon** [comment]
New url: http://pattern.getproudcity.com/?v=&page=dashboard

**Comment by lukefretwell** [comment]
How about the map for now and then we can circle back?
